### PR TITLE
fix: allow aliasname empty

### PR DESCRIPTION
**Pull request by banusha2**
## Context

User cannot rename the pipeline back to empty string, on changing AliasName to an empty string it gives `400`.

## Objective
Users can rename the pipeline alias back to an empty string.

## References
https://github.com/screwdriver-cd/screwdriver/issues/2744

## License

I confirm that this contribution is made under the terms of the license found in the root directory of this repository's source tree and that I have the authority necessary to make this contribution on behalf of its copyright owner.

### CODE REVIEW

File: config/settings.js [hunk 1]
```diff
@@ -38,6 +38,7 @@ const SCHEMA_PIPELINE_SETTINGS = Joi.object()
         showEventTriggers: Joi.boolean().optional(),
         filterEventsForNoBuilds: Joi.boolean().optional(),
         aliasName: Joi.string()
+            .allow('')
```
Comment (pritamstyz4ever) [inline config/settings.js:41]: does it need to be empty or optional settings?

**Comment by banusha2** [comment]
as per requirement it should be empty.

**Comment by banusha2** [comment]
it's an optional field but later users want to rename it to an empty string
they should be able to do .

**Comment by pritamstyz4ever** [comment]
> it's an optional field but later users want to rename it to an empty string they should be able to do . On Mon, Aug 8, 2022 at 1:27 PM Anusha Bollampally < ***@***.***> wrote:
> […](#)
> as per requirement it should be empty. 

Please add `.optional()` if it is an optional field

**Comment by banusha2** [comment]
Please check .optional() exist already

File: config/settings.js (hunk 1 above)
Comment (tkyi) [reply config/settings.js:41]: I see `.optional()` on line 45, so should be okay I think
just wondering if you mean `.allow(null)` vs `.allow('')`

**Review by pritamstyz4ever — APPROVED** [review]

File: config/settings.js (hunk 1 above)
Comment (banusha2) [reply config/settings.js:41]: We want user to be able change back aliasname to empty string, so I believe `.allow('')` will be good, what do you think on this?

**Review by tkyi — APPROVED** [review]

File: config/settings.js (hunk 1 above)
Comment (tkyi) [reply config/settings.js:41]: sure

**Pull request merged**